contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -181,6 +244,12 @@ to unbreak themselves (and then file a bug). If this happens, we must
 reconsider the deprecation and may choose to abandon it entirely by changing
 the gate back to `true` for a release or two and eventually removing it.
 
```

Comment:
Minor, but most of this doc is passive-voice, so I might rephrase it as

"""
Once the deprecation period has passed (link to deprecation policy?), the gate should be locked to the default value (`LockToDefault: true`).  As with GA features, all references to the feature gate must be kept for a minimum of three releases after the gate has been locked to the default value. The gate, all references to it, and all gated logic may be removed after those three releases.
"""

Maybe xref the compat-versions header?

Reply:
added